### ISSUE TITLE
Review translation policy

### ISSUE THREAD

**Issue by rylev** [rust-lang/surveys#192]
Translating the annual survey is a time consuming task that requires the coordination of many volunteers both before and after the annual survey is administered. We should consider which languages bring the most benefit and consider (with consultation from members of those language communities) whether it makes sense to not translate the survey into that language. As a second step, we may wish to try to find out if there are some language communities who largely did not participate in the survey because it was not offered in their language. 

While it might seem that the obvious approach is to keep languages with the highest number of respondents in the 2021 survey, we might be better off considering language communities which normally do not wish to communicate in English. 

For example, Simplified Chinese and German were the 2nd and 3rd most popular languages for response in the 2021 survey, but Japanese and Simplified Chinese included the most respondents who did not show a preference for technical communication in English. This points towards the possibility that we might get more use out of a Japanese translation than a German translation.

**Comment by YohDeadfall** [comment]
Will numbers of participants be published for each language including English?

**Comment by rylev** [comment]
These numbers are for "completed" surveys (i.e., the respondent saw every question and answered all required questions): 

* Chinese, Simplified: 594
* Chinese, Traditional: 49
* English: 7259
* French: 317
* German: 399
* Japanese: 184
* Korean: 73
* Portuguese: 75
* Russian: 283
* Spanish: 121

**Comment by apiraino** [comment]
This year we adopted an approach to translations based on "volunteers we could find" rathen than trying hard to represent languages based on popularity. For example @YohDeadfall volunteered for Russian (thanks!)

Yes, it's a time-consuming task but there's not a lot of ways around it. The tooling we're currently using also does not provide a great workflow for translations.

Closing as there's no more actionables at this point for this issue.
